main.tf

```diff
@@ -219,6 +219,13 @@ resource "azurerm_kubernetes_cluster" "main" {
       client_secret = var.client_secret
     }
   }
+  storage_profile {
```

Comment:
Could we make this block a `dynamic` block and add a new toggle variable so we can turn it on and off like we've done for [`aci_connector_linux`](https://github.com/Azure/terraform-azurerm-aks/blob/master/main.tf#L90-L96)?